lib/core/batchingReceiver.ts

```diff
@@ -31,11 +31,11 @@ export class BatchingReceiver extends MessageReceiver {
    * @param {ClientEntityContext} context The client entity context.
    * @param {ReceiveOptions} [options]  Options for how you'd like to connect.
    */
-  constructor(context: ClientEntityContext, options?: ReceiveOptions) {
-    super(context, ReceiverType.batching, options);
-  }
+	constructor(context: ClientEntityContext, options?: ReceiveOptions) {
```

Comment:
looks like indentation is set to 4 spaces instead of 2

Reply:
It is difficult to review the actual changes .Can you fix the indentation before the PR can be reviewed?

Reply:
Yes, just did.